rails/Gemfile

```diff
@@ -57,6 +57,7 @@ gem 'in_place_editing', git: 'https://github.com/concord-consortium/in_place_edi
 gem 'json', '~> 1.8.6'
 gem 'jwt'
 gem 'jquery-fileupload-rails'
+gem 'mimemagic', '~> 0.3.7'
```

Comment:
Perhaps this should be 0.3.10 since I think that is the latest version that supports the external database. There were a series of other releases before this which I'd guess had bugs or used different approaches.